docs/platform/server-v3/modules/about.md

```diff
@@ -14,100 +14,132 @@ enterprise_only: true
 - [WordPress module (blog)](../../integration/wordpress)
 - [Algolia search module (search)](../../integration/algolia)
 
-## Old vs new approach
+## What is a module in Falcon
 
-In previous version of Falcon Server (version 2) there was a distinct separation between extensions, api clients, rest endpoint handlers and event handlers. Each of these needed to be implemented and loaded as a separate package. Falcon Server 3 changes that approach and introduces concept of a module. 
+A module is a package that contains implementations of all the pieces required to achieve a particular thing. Each module needs to export its content via named exports, default exports will be ignored. Our middleware (Falcon Server) expects only known module extension classes (common services), anything else, which is outside of the predefined list will be ignored:
```

Comment:
Not clear what is the `predefined list` here, mb we can add it later with some examples
```suggestion
A module is a package that contains implementations of all the pieces required to achieve a particular thing. Each module needs to export its content via named exports, default exports will be ignored. Our middleware (Falcon Server) expects only known module extension classes (common services), everything else will be ignored:
```

Reply:
done